test/unit/Main.test.js

```diff
@@ -58,6 +62,36 @@ describe('Main', () => {
     });
   });
 
+  describe('Hash id on mount', () => {
+    let component;
+
+    it('should do nothing if the location is wrong', () => {
+      component = mount(
+        <Main
+          listOptions={config.staffPicksListOptions}
+          currentAudience="Adult"
+          location={{ hash: '#01-third-book-title' }}
+          picksData={{ picks }}
+          />
```

Comment:
Eslint complains this and line 88 about the closing bracket is not aligned with opening tag.